### PR TITLE
style(eslint-fixes): change eslint to standard and fix warnings

### PR DESCRIPTION
**Pull request by lcustodio**
This PR is just a proposition in order to enhance the code practices.

* Add prettier at dev dependency and as npm task
* Clean up of eslint conf (avoid conflict between eslint and prettier)

### CODE REVIEW

**Comment by danilowoz** [comment]
Hi @lcustodio, thanks for your contribution!
Actually I started to use the [prettier](https://github.com/prettier/prettier), you know? BUT I forgot to put the config file inside the project ¬¬

It is like this `.prettierrc`:
```json
{
  "printWidth": 100,
  "tabWidth": 2,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "jsxBracketSameLine": false,
  "parser": "babylon",
  "semi": false
}
```

Can you add it? What you think?

**Comment by lcustodio** [comment]
Ah great! Prettier is awesome. 👍 

* add prettier at dev dependency and as npm task
* clean up of eslint conf (avoid conflict between eslint and prettier)

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
   "scripts": {
     "watch": "babel --plugins transform-es2015-modules-umd src --watch --ignore tests --out-dir ./dist",
     "prepublish": "babel --plugins transform-es2015-modules-umd src --ignore tests --out-dir ./dist",
-    "lint": "eslint ./src",
-    "lintfix": "eslint ./src --fix",
+    "eslint": "eslint 'src/**/*.js'",
+    "prettier": "prettier --write 'src/**/*.{js,jsx,json,css}'",
     "testonly": "mocha $npm_package_options_mocha",
     "test": "npm run lint && npm run testonly",
```
Comment (danilowoz) [inline package.json]: @lcustodio you just need to update this script (from `lint` to `eslint`)

Comment (lcustodio) [reply package.json]: `lint` it is 😉

Comment (danilowoz) [reply package.json]: But you need to change in the script: `"test": "npm run lint && npm run testonly"`, you know?

Comment (lcustodio) [reply package.json]: Sorry, I wasn't clear. I reverted the script `eslint` back to `lint`, so the former `test` works. 

```json
"scripts": {
    "----> lint <----": "eslint ./src",
    "prettier": "prettier --write 'src/**/*.{js,jsx,json,css}'",
    "testonly": "mocha $npm_package_options_mocha",
    "test": "npm run ----> lint <---- && npm run testonly",
  }
```

Comment (danilowoz) [reply package.json]: Ahhh ok, I did't see your last commit 👍

**Comment by danilowoz** [comment]
Thanks @lcustodio! Feel free to contribute again! 🍺

**Pull request merged**